lib/Driver/Compilation.cpp

```diff
@@ -1163,7 +1186,12 @@ namespace driver {
           continue;
         }
 
-        // Can we run a cross-module incremental build at all? If not, fallback.
+        // Is this module out of date? If not, just keep searching.
```

Comment:
Not your fault, but `collectIncrementalExternallyDependentJobsFromDependencyGraph` is too long for my comfort, too.